### PR TITLE
upload ISS WDL

### PR DESCRIPTION
**Pull request by ambrosejcarr**
* Adds a workflow for processing ISS data using the starfish python API
* Adds inputs that trigger test-sized data and the complete spaceTx ISS mouse dataset.

### CODE REVIEW

File: workflows/iss/iss.wdl [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+        # blobs = dots; define the spots in the dots image, but then find them again in the stack.
+        intensities = p.run(filtered_imgs, blobs_image=dots, blobs_axes=(Axes.ROUND, Axes.ZPLANE))
+
+        # decode
+        decoded = experiment.codebook.decode_per_round_max(intensities)
+
+        # save results
+        df = decoded.to_decoded_spots()
+        df.save_csv("decoded.csv")
```
Comment (shanaxel42) [inline workflows/iss/iss.wdl]: Do you wanna save the file as like "decoded.csv+ fov" so we know which fov the output is for?

File: workflows/iss/iss.wdl [hunk 2]
```diff
@@ -0,0 +1,134 @@
+
+task process_field_of_view {
+
+    String experiment
+    Int field_of_view
+
+    command <<<
+        python3 <<CODE
+
+        import numpy as np
```
Comment (ttung) [inline workflows/iss/iss.wdl]: move this into a .py file so we can test it.

even better, but entirely optional, would be to structure it as a recipe (rationale: other people would be less tempted to tinker with the pipeline in a way that makes it non-traceable from a provenance POV).

File: workflows/iss/iss.wdl [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+        df = decoded.to_decoded_spots()
+        df.save_csv("decoded.csv")
+        CODE
+    >>>
+
+    runtime {
+        docker: "spacetx/starfish:0.1.0-simple"
+        memory: "16 GB"
+        cpu: "4"
+        disk: "local-disk 100 SDD"
```
Comment (ttung) [inline workflows/iss/iss.wdl]: I suspect you don't need a 100GB SSD to process this dataset.  There's literally nothing here that takes advantage of disk.

You may also want some config to disable the slicedimage caching.  Rationale: You will not load the data twice, so writing it to disk is counterproductive.

**Comment by ambrosejcarr** [comment]
@shanaxel42 had some ideas about how to follow up on @ttung 's requests, so I'm going to reassign this to avoid duplicating work.

**Comment by shanaxel42** [comment]
> @shanaxel42 had some ideas about how to follow up on @ttung 's requests, so I'm going to reassign this to avoid duplicating work.

I think this work is taken care of in https://github.com/spacetx/starfish/pull/1364/files. So I'm going to close this one out!

**Pull request closed without merging**